src/components/syncPlay/ui/playbackPermissionManager.js

```diff
@@ -32,8 +34,12 @@ class PlaybackPermissionManager {
      * Tests playback permission. Grabs the permission when called inside a click event (or any other valid user interaction).
      * @returns {Promise} Promise that resolves succesfully if playback permission is allowed.
      */
-    check () {
-        return new Promise((resolve, reject) => {
+    async check () {
```

Comment:
async/await doesn't seem necessary here. I think you could just return `Promise.resolve(true)` below and remove the await when returning the promise.